SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
```

Comment:
That's not the current state, see https://github.com/eclipse-tractusx/portal/blob/main/SECURITY.md

Reply:
Addressed with : https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/1/commits/13ee3d5d9d48f529197d16e1e45ad6af827111b9